src/main/java/com/spotify/autoscaler/api/KCCAutoscalerController.java

```diff
@@ -59,66 +59,86 @@ public KCCAutoscalerController(final Database database) {
 
   @POST
   @Path("/reconcile")
```

Comment:
Can we remove the `/kcc` above and just leave reconcile? Alternatively rename kcc to declarative, also in the class name.